repos/fdbt-site/src/pages/products/exports.tsx

```diff
@@ -234,11 +236,37 @@ export const getServerSideProps = async (ctx: NextPageContextWithSession): Promi
     if (exportStarted) {
         updateSessionAttribute(ctx.req, SELECT_EXPORTS_ATTRIBUTE, undefined);
     }
+    const exportNames = await getS3Exports(noc);
+
+    const exports = await Promise.all(
+        exportNames.map(async (name) => {
+            const prefix = `${noc}/exports/${name}/`;
+
+            const matchingDataCount = await getS3FolderCount(MATCHING_DATA_BUCKET_NAME, prefix);
+            const netexCount = await getS3FolderCount(NETEX_BUCKET_NAME, prefix);
+
+            let numberOfFilesExpected = matchingDataCount;
+
+            let metadata = undefined;
+            const metaDataExists = await checkIfMetaDataExists(`${noc}/exports/${name}.json`);
+
+            if (metaDataExists) {
+                metadata = await getExportMetaData(`${noc}/exports/${name}.json`);
```

Comment:
The problem with this approach, is that we will figure out if an export is in progress, and pass that through to the page as a prop (true or false).

And then that prop will never be updated in the page, because its not using state. 

We need a line of code above like this:

const [exportInProgress, setExportInProgress] = useState(exportInProgress);

So when the page loads, it will have the default value from the page props, and then we will need some code which updates this value.